docs/dbt/run-dbt-spark.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: "Run dbt-spark against a pySpark session."
+sidebarTitle: "Run dbt-spark"
+---
+
+The following procedure demonstrates how to run `dbt-spark` with a pySpark session.
+
+1.  Start the Mage development environment through the following command:
+```
+$  ./scripts/dev.sh demo_project 
+```
+2. Click the `Terminal` icon on the right side of the Mage UI, and install the
```

Comment:
Let's not use Terminal to install the packages since it's one-off. Every time you restart the container, the packages installed via terminal are not persisted.
You can build custom docker image with spark libraries installed like this: https://docs.mage.ai/integrations/spark-pyspark#1-build-mage-docker-image-with-spark-environment

Reply:
Good catch: updated the procedure to use a custom docker image.